src/main.rs

```diff
@@ -475,7 +482,7 @@ fn run() -> Result<(), failure::Error> {
         let env = matches.value_of("env");
         let mut target = manifest.get_target(env)?;
 
-        commands::publish(&user, &mut target)?;
+        commands::publish(&user, &mut target, matches.is_present("print-files"))?;
```

Comment:
nit: Could we separate this `matches.is_present` to a separate line?

Reply:
i already cargo fmtted this so this should be fine?